delta/sourcing-psql/src/test/scala/ch/epfl/bluebrain/nexus/delta/sourcing/projections/FailedElemLogStoreSuite.scala

```diff
@@ -143,8 +147,22 @@ class FailedElemLogStoreSuite extends BioSuite with MutableClock.Fixture with Do
     assertList(project1, projection12, FromPagination(1, 2), TimeRange.Anytime, List(fail3, fail4))
   }
 
+  private val after = TimeRange.After(fail3.instant)
+  test(s"List failures after ${fail3.instant} for ${metadata12.name}") {
+    assertList(project1, projection12, Pagination.OnePage, after, List(fail3, fail4))
+  }
+
+  test(s"Count failures after ${fail3.instant} for ${metadata12.name}") {
+    store.count(project1, projection12, after).assert(2L)
+  }
+
+  private val before = TimeRange.Before(fail3.instant)
   test(s"List failures before ${fail3.instant} for ${metadata12.name}") {
-    assertList(project1, projection12, Pagination.OnePage, TimeRange.Before(fail3.instant), List(fail2, fail3))
+    assertList(project1, projection12, Pagination.OnePage, before, List(fail2, fail3))
+  }
+
+  test(s"Count failures before ${fail3.instant} for ${metadata12.name}") {
+    store.count(project1, projection12, before).assert(2L)
```

Comment:
These test names seem too fixture-specific. Isn't the important thing (giving the first test as an example) that's you're listing failures of a certain projection after a certain time? The specific time and metadata (which isn't actually referred to in the test) doesn't help me understand what the test is supposed to do, it's actually making it harder